engine/config/1.0/schema.json

```diff
@@ -18,28 +18,32 @@
         "addr": { "type": "string" },
         "password": { "type": "string" },
         "db": {
+          "default": 0,
+          "description": "The database to use for the Redis connection.",
           "oneOf": [
             { "type": "integer" },
             { "$ref": "#/$defs/envVarPattern" },
             { "$ref": "#/$defs/filePattern" }
           ]
         },
         "read_timeout": {
+          "description": "The read timeout for the Redis connection in seconds.",
           "oneOf": [
             { "type": "integer" },
             { "$ref": "#/$defs/envVarPattern" },
             { "$ref": "#/$defs/filePattern" }
           ]
         },
         "write_timeout": {
+          "description": "The write timeout for the Redis connection in seconds.",
           "oneOf": [
             { "type": "integer" },
             { "$ref": "#/$defs/envVarPattern" },
             { "$ref": "#/$defs/filePattern" }
           ]
         }
       },
-      "required": ["addr", "db"],
+      "required": ["addr"],
```

Comment:
`db` is not actually required; the Engine will use `0` as a default value if `db` is omitted.